CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Aloxide
+
+1. [Getting Involved](#getting-involved)
+2. [How To Develop Features](#how-to-develop)
+3. [How To Report Bugs](#how-to-report-bugs)
+
+## Getting Involved
+
+We're always looking for help [implementing features](#how-to-develop), [identifying bugs](#how-to-report-bugs), writing test cases, and improving documentation. If you find any issues, please don't hesitate to discuss with us, any ideas are helps for us.
+
+There're some related guides for you to check before contributing to our project:
+- [High-level Architecture](https://github.com/lecle/aloxide/wiki/High-level-Architecture)
+- [Commit convention]()
```

Comment:
The link is missing?

Reply:
I was adding commit convention to the wiki but remembering that we don't discuss about this convention yet, I'm going to remove it.